libHttpClient.vs2019.sln

```diff
@@ -37,19 +33,13 @@ Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "libcrypto.142.Win32", "Buil
 EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "libssl.142.Win32", "Build\libssl.142.Win32\libssl.142.Win32.vcxproj", "{13457617-8476-4708-A601-AF998F6EDF21}"
 EndProject
-Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "libssl.142.Android", "Build\libssl.142.Android\libssl.142.Android.vcxproj", "{F2FE27E9-898D-47DD-AC6A-802680293ECE}"
-EndProject
-Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "libcrypto.142.Android", "Build\libcrypto.142.Android\libcrypto.142.Android.vcxproj", "{73B6046F-BB41-4067-A5CC-FC540A1C2CAE}"
-EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "Win32WebSocket2019", "Samples\Win32WebSocket\Win32WebSocket2019.vcxproj", "{9A461116-4B40-40F0-8AE0-81DD294BB92E}"
 EndProject
-Project("{39E2626F-3545-4960-A6E8-258AD8476CE5}") = "libHttpClient.142.Android.Java", "Build\libHttpClient.142.Android.Java\libHttpClient.142.Android.Java.androidproj", "{A91A390A-C4F0-40A9-A519-27FFDD20DEC2}"
-EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "libHttpClient.142.XDK.C", "Build\libHttpClient.142.XDK.C\libHttpClient.142.XDK.C.vcxproj", "{9E0EB8C2-40CD-448F-9B98-DAF9830EF9AD}"
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "GDK", "GDK", "{9C8A0D94-F592-46D7-9491-1E464C6A8166}"
 EndProject
-Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "libHttpClient.142.GDK.C", "Build\libHttpClient.142.GDK.C\libHttpClient.142.GDK.C.vcxproj", "{DA74F6C1-9B77-4135-9C23-AA348AB0DC9C}"
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "libHttpClient.142.GDK.C", "Build\libHttpClient.142.GDK.C\libHttpClient.142.GDK.C.vcxproj", "{80D8061D-BBEF-414D-8F08-4D8B52C02E85}"
 EndProject
```

Comment:
libhttpclient.props says

      <Project Condition="'$(HCLibPlatformType)' == 'GDK' and $(HCLibToolset)==141">{DA74F6C1-9B77-4135-9C23-AA348AB0DC9C}</Project>

and you removed this id

Reply:
whoops. should be reset now!